tests/test_utils.py

```diff
@@ -648,12 +649,22 @@ def test_psr_utils():
 
     # Mass function
```

Comment:
Can y ou make each of these a separate test, so we can tell what broke and what didn't?

Reply:
I was just adding to how it was before, but I agree it seems odd.  I'll break out at least some of these.